aspnetcore/fundamentals/openapi/aspnetcore-openapi.md

```diff
@@ -239,10 +239,12 @@ Create a new ASP.NET Core Web API (Native AOT) project.
 dotnet new webapiaot
 ```
 
+:::moniker range="< aspnetcore-10.0"
```

Comment:
Problems:
1. you can't have nested monikers
2. It's currently set to `:::moniker range=">= aspnetcore-9.0"`
```suggestion
```

Reply:
Hi @Rick-Anderson !

Thanks for the quick review.
I've made some changes to prevent nested monikers. 

The goal is to ONLY show the "add package and add this code" for .NET 9, as .NET 10 will not need this anymore. This is the way I've now written it.